<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
font-size: 15px;
font-weight: bold;
border-radius: 20px;
background-color: #73C2FB;
color: black;
border: 2px solid rgba(0, 0, 0, 60);
padding: 15px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txt_guess">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    font-size: 15px;
    font-weight: bold;
    border-radius: 20px;
    background-color: #73C2FB;
    color: black;
    border: 2px solid rgba(0, 0, 0, 100);
    padding: 15px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_guess">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 15px;
    font-weight: bold;
    border-radius: 20px;
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                      stop:0 #FFFF00, stop:0.5 white);
    color: black;
    border: 2px solid rgba(0, 0, 0, 100);
    padding: 15px;
}</string>
    </property>
    <property name="text">
     <string>Guess</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_instruction">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 15px;
    font-weight: bold;
    border-radius: 20px;
    background-color:#FFFFE0 ;
    color: black;
    border: 2px solid rgba(0, 0, 0, 60);
    padding: 15px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#73C2FB;</string>
    </property>
    <property name="text">
     <string>Guess Number Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_restart">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 15px;
    font-weight: bold;
    border-radius: 20px;
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                      stop:0 #73C2FB, stop:1 white);
    color: black;
    border: 2px solid rgba(0, 0, 0, 100);
    padding: 15px;
}</string>
    </property>
    <property name="text">
     <string>New Target</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_min_num">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    font-size: 15px;
    font-weight: bold;
    border-radius: 20px;
    background-color: #73C2FB;
    color: black;
    border: 2px solid rgba(0, 0, 0, 100);
    padding: 15px;
}</string>
    </property>
    <property name="text">
     <string>         1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_max_num">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    font-size: 15px;
    font-weight: bold;
    border-radius: 20px;
    background-color: #73C2FB;
    color: black;
    border: 2px solid rgba(0, 0, 0, 100);
    padding: 15px;
}</string>
    </property>
    <property name="text">
     <string>      100</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>503</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
